src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2023,6 +2048,30 @@ components:
           type: string
           example:
             "Main group"
+    UserGroupMember:
+      type: object
+      properties:
+        user_pk:
+          type: integer
+          description: user's primary key
+          example: 1
+        user_name:
+          type: string
+          description: user name
+        user_status:
+          type: string
+          description: Status of the user
+        user_desc:
+          type: string
+          description: Description of the user
+        group_user_member_pk:
+          type: integer
+          description: Group user member primary key.
+          example: 2
```

Comment:
This is an internal key and never used by any API. IMO, it should not be exposed by the API as well.

Reply:
But I think the frontend is going to need it when requesting for the updates on the specific membership roles.
What do you think ?

Reply:
I don't see it anywhere being used in the existing code. So for sure it'll not be needed by the UI you are working on.

And in case it's ever needed, we can expose it then.